### DISCUSSION TITLE
OpenTelemetry Support

### DISCUSSION THREAD

**Discussion by yossicohn-hs** [argoproj/argo-workflows#11218] [Q&A] [answered]
Hi, a question regarding Workflow observability.
Is there a way to use OpenTelemetry with Argo workflows?

**Comment by agilgur5** [comment, marked as answer]
Yes, Argo emits a variety of [metrics](https://argoproj.github.io/argo-workflows/metrics/) that you can customize.

If you are asking about distributed tracing more specifically, there is an open feature request for that in #12077
